src/Data/AVL/Map.agda

```diff
@@ -0,0 +1,94 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Maps from keys to values, based on AVL trees
+-- This modules provides a simpler map interface, without a dependency
+-- between the key and value types.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary using (StrictTotalOrder)
+
+module Data.AVL.Map
+  {a ℓ₁ ℓ₂} (strictTotalOrder : StrictTotalOrder a ℓ₁ ℓ₂)
+  where
+
+open import Data.Bool.Base using (Bool)
+open import Data.List.Base using (List)
+open import Data.Maybe using (Maybe)
+open import Data.Product using (_×_)
+open import Level using (_⊔_)
+
+import Data.AVL strictTotalOrder as AVL
+open StrictTotalOrder strictTotalOrder renaming (Carrier to Key)
+
+------------------------------------------------------------------------
+-- The map type
+
+Map : ∀ {i} → (v : Set i) → Set (a ⊔ i ⊔ ℓ₂)
```

Comment:
We usually mark types using capitals, so I think it would be more consistent to call the universe level `v` and the set `V : Set v` so the type would become:
```
Map : ∀ {v} (V : Set v) → Set (a ⊔ v ⊔ ℓ₂)
```
Otherwise looks great!

Reply:
Thanks, I updated the PR according to this suggestion.